### ISSUE TITLE
PSA hierarchical storage: get/set/remove

### ISSUE THREAD

**Issue by gilles-peskine-arm** [ARMmbed/mbedtls#6364]
Implement individual file access functions for [PSA hierarchical storage](https://github.com/Mbed-TLS/mbedtls/issues/6363): `get`, `get_info`, `set`, `remove`.

Deliverables: implementation of these functions, and unit tests. Using the new API outside of its own unit tests is out of scope.

Prerequisites: https://github.com/Mbed-TLS/mbedtls/issues/6162, https://github.com/Mbed-TLS/mbedtls/issues/6363
